httpie/manager/__main__.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import sys
+
+from typing import List, Union
+
+from httpie.context import Environment
+from httpie.manager.plugins import PluginInstaller
+from httpie.status import ExitStatus
+from httpie.manager.cli import parser
+
+
+MSG_COMMAND_CONFUSION = '''
+This command is only for managing HTTPie plugins.
+To send a request, please use the http/https commands:
+
+  $ http {args}
+
+  $ https {args}
+
+'''
+
+# noinspection PyStringFormat
+MSG_NAKED_INVOCATION = f'''\
+Please specify one of these: "plugins"
+
+{MSG_COMMAND_CONFUSION}
+
+'''.format(args='POST pie.dev/post hello=world')
```

Comment:
This is still not very clear to me as auser who runs `httpie`, it should rather display the help message of `httpie` including the confusion paragraph at the starting.

Reply:
What do you mean by that? We already print both the usage as well as the confusion paragraph.
```
 $ httpie
usage: httpie [-h] [--debug] [--traceback] {plugins} ...
httpie: error: Please specify one of these: "plugins"


This command is only for managing HTTPie plugins.
To send a request, please use the http/https commands:

  $ http POST pie.dev/post hello=world

  $ https POST pie.dev/post hello=world
```

Reply:
I am saying we can directly just sent the output of `--help` if no arguments are specified because it sums the two messages (above) pretty well and it was more clear for me when looking at them.